apps/enc.c

```diff
@@ -586,7 +586,7 @@ int enc_main(int argc, char **argv)
     if (benc != NULL)
         wbio = BIO_push(benc, wbio);
 
-    for (;;) {
+    for (inl = 0; !BIO_eof(rbio);) {
```

Comment:
same here, inl is not used after this loop.
so please `while (!BIO_eof(rbio)) {`

Reply:
done